jobsdb/jobsdb.go

```diff
@@ -1467,6 +1469,7 @@ func (jd *Handle) createDSTablesInTx(ctx context.Context, tx *Tx, newDS dataSetT
 		custom_val VARCHAR(64) NOT NULL,
 		event_payload `+string(columnType)+` NOT NULL,
 		event_count INTEGER NOT NULL DEFAULT 1,
+		payload_size INTEGER NOT NULL DEFAULT 0,
```

Comment:
allowing `NULL` would let us query right away by `sum(COALESCE(jobs.payload_size, pg_column_size(jobs.event_payload))) over (order by jobs.job_id) as running_payload_size`?

Reply:
but we are not doing a COALESCE now, right?

Reply:
We aren't yeah.
I just went with the 2 step solution.
Thought of this afterwards.